docs/notebooks/proton_radiography_particle_tracing_custom_source.ipynb

```diff
@@ -0,0 +1,524 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Proton Radiography: Synthetic Radiographs with Custom Source Profiles"
```

Comment:
Can you shorten the title by at least 1 word?  Currently the title is so long that it's extending beyond the boundaries of the thumbnail.

<img width="709" alt="image" src="https://user-images.githubusercontent.com/29869348/110225821-c9784600-7e9d-11eb-9792-894c8f25eff7.png">

Reply:
Sure, let's try just "Synthetic Radiographs with Custom Source Profiles"